### DISCUSSION TITLE
how to exclude all tailwindcss base css and only export what  i used

### DISCUSSION THREAD

**Discussion by coader** [tailwindlabs/tailwindcss#10586] [Help]
because I use it in component library, so if the exported css include base selector like this:
button,[type=button],[type=reset],[type=submit]{-webkit-appearance:button;background-color:transparent;background-image:none}

will be repeated and confuse the ordinal of all other imported css

**Comment by wongjn** [comment]
You can [disable the preflight core plugin](https://tailwindcss.com/docs/preflight#disabling-preflight).

**Reply by coader** [reply]
> You can [disable the preflight core plugin](https://tailwindcss.com/docs/preflight#disabling-preflight).

tks, I also help to trim that variables:
![image](https://user-images.githubusercontent.com/3700498/219828231-4454c6ff-ab3e-4006-a481-db4d7bbf0a9b.png)
